src/index.html

```diff
@@ -7,11 +7,167 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     class="form_field"
```

Comment:
wrong identation, add 2px
```suggestion
       action="https://mate-academy-form-lesson.herokuapp.com/create-application"
       method="post"
       class="form_field"
```